cc/utils/cose/cwt.cc

```diff
@@ -84,4 +86,23 @@ absl::StatusOr<Cwt> Cwt::Deserialize(absl::string_view data) {
   return Cwt(iss->asTstr(), sub->asTstr(), std::move(*subject_public_key), std::move(item));
 }
 
+absl::StatusOr<std::vector<uint8_t>> Cwt::SerializeHpkePublicKey(
+    const std::vector<uint8_t>& public_key) {
+  auto serialized_public_key_certificate = CoseKey::SerializeHpkePublicKey(public_key);
```

Comment:
ASSIGN_OR_RETURN

(if this macro exists)

Reply:
Macro don't exist in the opensource =(